### PR TITLE
Docs/styling docs 243 update

### PR DESCRIPTION
**Pull request by rolfsmeds**
Updated styling documentation for various components
- Added border props to input field props reference
- Added new props for AppLayout, Grid, TreeGrid, MSCB, RTE
- Fixed duplicate UUID in SplitLayout page

### CODE REVIEW

File: articles/components/multi-select-combo-box/styling.asciidoc [hunk 1]
```diff
@@ -15,6 +15,40 @@ include::../_styling-section-theming-props.adoc[tag=label-helper-error]
...
+|===
+
+
+=== Miscellaneous
+
+[cols="1,2,2"]
+|===
+| Feature | Property | Default Value
+
+|Overlay width
```
Comment (web-padawan) [inline articles/components/multi-select-combo-box/styling.asciidoc:41]:
```suggestion
|Chip min width
```

File: articles/components/multi-select-combo-box/styling.asciidoc [hunk 2]
```diff
@@ -15,6 +15,40 @@ include::../_styling-section-theming-props.adoc[tag=label-helper-error]
...
+|`--vaadin-multi-select-combo-box-chip-min-width`
+|`50px`
+
+|Overlay width
```
Comment (web-padawan) [inline articles/components/multi-select-combo-box/styling.asciidoc:45]:
```suggestion
|Input min width
```

File: articles/components/multi-select-combo-box/styling.asciidoc (hunk 1 above)
Comment (rolfsmeds) [reply articles/components/multi-select-combo-box/styling.asciidoc:41]: Whops, thanks, that was sloppy of me.

**Review by web-padawan — APPROVED** [review]

**Review by web-padawan — COMMENTED** [review]
There is a merge conflict in `articles/components/multi-select-combo-box/styling.asciidoc`.

**Pull request merged**